mods/fire/init.lua

```diff
@@ -249,21 +259,16 @@ else
 		end,
 	})
 
-	-- Remove basic flames and flammable nodes
+	-- Remove flammable nodes
 
 	minetest.register_abm({
 		nodenames = {"fire:basic_flame"},
 		interval = 5,
 		chance = 6,
 		catch_up = false,
 		action = function(p0, node, _, _)
-			-- If there are no flammable nodes around flame, remove flame
 			local p = minetest.find_node_near(p0, 1, {"group:flammable"})
-			if not p then
-				minetest.remove_node(p0)
-				return
-			end
-			if math.random(1, 3) == 1 then
+			if p and math.random(1, 3) == 1 then
```

Comment:
As we discussed earlier, this chance should be combined into the ABM chance (it becomes chance = 18), so the ABM runs 3 times less often.